production/dea_ml/docker/Dockerfile

```diff
@@ -0,0 +1,174 @@
+ARG V_BASE=-3.0.4
+ARG py_env_path=/env
+
+FROM opendatacube/geobase:wheels${V_BASE} as env_builder
+ARG py_env_path
+
+RUN mkdir -p /conf
+
+# Some libs assume presence of Cython and numpy in the environment, so build
+# minimal env with those first, then extend further
+COPY requirements0.txt /conf/
+RUN echo "Building python environment: stage 1" \
+  && env-build-tool new /conf/requirements0.txt ${py_env_path}
+
+COPY requirements.txt /conf/
+RUN echo "Building python environment: stage 2" \
+  && env-build-tool extend /conf/requirements.txt ${py_env_path}
+
+ENV PATH=${py_env_path}/bin:$PATH
+
+# COPY requirements-jupyter.txt /conf/
+# RUN echo "Building python environment: stage Jupyter" \
+#   && env-build-tool extend /conf/requirements-jupyter.txt ${py_env_path}
+#
+# COPY jupyter-extensions.txt /conf/
+# # TAB_NAME -- name to appear in the browser tab
+# ARG TAB_NAME="DEA Sandbox"
+# RUN echo "Adding jupyter lab extensions" \
+#   && while read ext; do \
+#       if [ -n "${ext}" ]; then \
+#         echo "Installing ${ext}"; \
+#         jupyter labextension install --no-build "${ext}"; \
+#       fi; \
+#      done < /conf/jupyter-extensions.txt \
+#   && jupyter lab build --name="${TAB_NAME}" \
+#   && jupyter lab clean \
+#   && jupyter labextension list \
+#   && echo "...done"
+#
+# RUN echo "Enable server extensions" \
+#   && jupyter serverextension enable --py --sys-prefix nbresuse \
+#   && jupyter serverextension enable --py --sys-prefix jupyterlab_code_formatter \
+#   && jupyter serverextension enable --py --sys-prefix jupyterlab_iframe \
+#   && echo done
+
+COPY requirements-odc-static.txt /conf/
+RUN echo "Adding odc-dependencies (static)" \
+  && env-build-tool extend /conf/requirements-odc-static.txt ${py_env_path}
+
+FROM opendatacube/geobase:runner${V_BASE}
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+#Install ffmpeg static binary
+RUN apt-get update -y \
+  && apt-get install -y --fix-missing --no-install-recommends xz-utils curl \
+  && rm -rf /var/lib/apt/lists/* \
+  && curl -s -L https://www.johnvansickle.com/ffmpeg/old-releases/ffmpeg-4.2.2-amd64-static.tar.xz > ffmpeg.tar.xz \
+  && echo "c1c7ec5180523db214a7f6f59ea737e8e582570cb884c3ab0c583da4f5165e27" ffmpeg.tar.xz | sha256sum -c - \
+  && tar xvJ --strip-components=1 -C /tmp < ffmpeg.tar.xz \
+  && mkdir -p /usr/local/bin/ \
+  && mv /tmp/ffmpeg /usr/local/bin/ffmpeg \
+  && rm -rf /tmp/* ffmpeg.tar.xz
```

Comment:
delete

Reply:
is that affect the tif read and write ?

Reply:
I thought ffmeg is for video and audio files